examples/simple/src/posts/PostCreate.js

```diff
@@ -132,10 +132,7 @@ const PostCreate = ({ permissions, ...props }) => {
                 <FormSpy subscription={{ values: true }}>
                     {({ values }) =>
                         values.title ? (
-                            <NumberInput
-                                source="average_note"
-                                defaultValue={5}
-                            />
+                            <NumberInput source="average_note" />
```

Comment:
Why have you removed the default value?

Reply:
Because final-form latest version doesn't take defaultValue into account if the initialValue is already defined at the form level (see https://github.com/final-form/final-form/pull/282)